Python/Product/PythonTools/PythonTools/Navigation/Peek/PythonPeekResultSource.cs

```diff
@@ -49,43 +49,26 @@ internal sealed class PythonPeekResultSource : IPeekResultSource {
             var fileName = PathUtils.GetFileOrDirectoryName(location.FilePath);
 
             var displayInfo = new PeekResultDisplayInfo2(
-                label: string.Format("{0} - ({1}, {2})", fileName, location.Line, location.Column),
+                label: string.Format("{0} - {1}", fileName, location.Span.Start),
                 labelTooltip: location.FilePath,
                 title: fileName,
                 titleTooltip: location.FilePath,
                 startIndexOfTokenInLabel: 0,
                 lengthOfTokenInLabel: 0
             );
 
-            int startLine = location.Line - 1;
-            int startColumn = location.Column - 1;
-            int endLine = startLine;
-            int endColumn = startColumn;
-            if (location.DefinitionStartLine.HasValue) {
-                startLine = location.DefinitionStartLine.Value - 1;
-            }
-            if (location.DefinitionStartColumn.HasValue) {
-                startColumn = location.DefinitionStartColumn.Value - 1;
-            }
-            if (location.DefinitionEndLine.HasValue) {
-                endLine = location.DefinitionEndLine.Value - 1;
-            }
-            if (location.DefinitionEndColumn.HasValue) {
-                endColumn = location.DefinitionEndColumn.Value - 1;
-            }
-
             return _peekResultFactory.Create(
                 displayInfo,
                 default(ImageMoniker),
                 location.FilePath,
-                startLine,
-                startColumn,
-                endLine,
-                endColumn,
-                location.Line - 1,
-                location.Column - 1,
-                location.Line - 1,
-                location.Column - 1,
+                (location.DefinitionSpan?.Start.Line ?? location.Span.Start.Line) - 1,
```

Comment:
you can introduce `var definitionSpan = location.DefinitionSpan ?? location.Span)` and simplify code by getting rid of null checks